.mbedignore

```diff
@@ -2,9 +2,8 @@ cryptoauthlib/app/secure_boot/
 cryptoauthlib/app/ip_protecton/
 cryptoauthlib/docs/
 cryptoauthlib/python/
+cryptoauthlib/test/*
```

Comment:
It looks like other places emit the `*` when excluding entire folders. I believe as written, this will not ignore invisible files in the test directory. (You'd need `cryptoauthlib/test/.*` for that.)

Reply:
@Patater I think that both test/* and test/ are the same according to this https://os.mbed.com/docs/mbed-os/v5.10/tools/mbed-os-build-rules.html#mbedignore
Anyway I rechecked this and  both of the solutions work fine.